src/server/api/AdminPanel.ts

```diff
@@ -0,0 +1,131 @@
+import type { CustomRequest } from "../../types.js";
+import type { Response } from "express";
+// @ts-ignore
+import { deleteUser, getMemberDataByCriteria, updateMemberColumns } from "../database/memberManager.js";
+// @ts-ignore
+import { deleteRefreshTokensOfUser } from "../database/refreshTokenManager.js";
+// @ts-ignore
+import { logEvents } from "../middleware/logEvents.js";
+
+async function processCommand(req: CustomRequest, res: Response): Promise<void> {
+	const command = req.params["command"]!;
+	const commandAndArgs = command.split(" ");
+	if (!req.memberInfo.signedIn) {
+		res.status(401).send("Cannot send commands while logged out.");
+		return;
+	}
+	if (!(req.memberInfo.roles?.includes("admin") ?? false)) {
+		res.status(403).send("Cannot send commands without the admin role");
+		return;
+	}
+	// TODO prevent affecting accounts with equal or higher roles
+	switch (commandAndArgs[0]) {
+		case "ban":
+			return;
+		case "delete":
+			if (commandAndArgs.length < 2) {
+				res.status(400).send("Invalid number of arguments, expected 1, got " + (commandAndArgs.length - 1) + ".");
+				return;
+			}
+			deleteUser(getMemberDataByCriteria(["user_id"], "username", commandAndArgs[1])["user_id"], commandAndArgs[2] ?? "");
+			res.status(200).send("");
+			logEvents("Command executed: " + command + "\n", "adminCommands");
```

Comment:
logEvents() automatically appends new lines.

Reply:
I want a double newline between entries, since an entry may span multiple lines.

Reply:
Okay, that's fine. I wasn't sure if it was intended or not.